Codigos/Estruturas-de-Dados/XorTrie/XorTrie.cpp

```diff
@@ -0,0 +1,36 @@
+struct XorTrie {
+    const int bits = 30;
+    vector<vector<int>> go;
+    int root = 0, cnt = 1;
+    XorTrie() { }
+    XorTrie(int n) { go.assign((n + 1) * bits, vector<int>(2, -1)); }
```

Comment:
```suggestion
    void build(int n) { go.assign((n + 1) * bits, vector<int>(2, -1)); }
```
This is the pattern we use on other structures